src/Components/TwoOneOneComponents/TwoOneOneHeader/TwoOneOneHeader.js

```diff
@@ -0,0 +1,228 @@
+import { useContext, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+import { AppBar, MenuItem, Select, Modal, Link, IconButton } from '@mui/material';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { Context } from '../../Wrapper/Wrapper';
+import twoOneOneMFBLogo from '../../../Assets/twoOneOneMFBLogo.png';
+import twoOneOneLinks from '../../../Assets/twoOneOneLinks';
+import LanguageIcon from '@mui/icons-material/Language';
+import ShareIcon from '@mui/icons-material/Share';
+import SaveAltIcon from '@mui/icons-material/SaveAlt';
+import MenuIcon from '@mui/icons-material/Menu';
+import Drawer from '@mui/material/Drawer';
+import Share from '../../Share/Share';
+import CloseIcon from '@mui/icons-material/Close';
+import EmailResults from '../../EmailResults/EmailResults';
+import './TwoOneOneHeader.css';
+
+const TwoOneOneHeader = ({ handleTextfieldChange }) => {
+  //this is so that when the users click on the cobranded logo, they're navigated back to step-0
+  const context = useContext(Context);
+  const { formData } = context;
+  const { urlSearchParams } = formData;
```

Comment:
Url search params are not stored in the backend, so if the page is refreshed, we won't be able to send the user to the right page if they click the home button. I think using the `formData.referrerCode` would be better for keeping the user on the right code.